Audio/AudioInput.cs

```diff
@@ -22,7 +22,8 @@ public class AudioEvent {
   public class AudioInput : Input {
 
     private static readonly int fftSize = 32768;
-    private static readonly int audioFormatSampleFrequency = 44100;
+    // Read this value from the sound card later.
+    private static int audioFormatSampleFrequency = -1;
```

Comment:
You should delete this variable as it's no longer getting used

Reply:
I'd like to do that, but it's not so simple because `private static double GetFrequencyBinUnrounded(double frequency)` is `static` and doesn't have access to the `device.AudioClient`. If we mark that function as non static, there are a large number of cascading changes that need to be made, and things get much more complicated.

Reply:
Ah, I was misled by GitHub search that only showed two locations it's used in.

![image](https://www.dropbox.com/s/8e7pzwye76pwfv4/Screenshot%202019-08-20%2017.33.44.png?dl=0)

You're right this will require more substantive changes. That said I'd prefer not to commit this hack. Can't prioritize this now but will try to figure it out later